README.md

```diff
@@ -137,6 +137,17 @@ The default value for `requriedConfig.url` is `vs/loader.js`.
 
 > You may need to note the [cross domain case](https://github.com/Microsoft/monaco-editor#integrate-cross-domain).
 
+### Using with @timkendrick/monaco-editor
+
+If you have a node-like browser environment that conflicts with Monaco's
+loader, you may have more luck with
```

Comment:
Please make "you may have more luck with" more specific. What exactly does the fork do differently and why does it promise success? What makes the other editor compatible with CommonJS? It is important to explain these things for people who are not familiar with all the JS stuff.

Reply:
Thanks, I made it a bit clearer.